closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -992,6 +1046,11 @@ private final class EngineAEADEnc implements ChaChaEngine {
             counter = 1;
         }
 
+        @Override
+        public int getOutputSize(int inLength, boolean isFinal) {
+            return (isFinal ? Math.addExact(inLength, TAG_LENGTH) : inLength);
```

Comment:
Please omit the outermost parentheses.

Reply:
This code matches what is in the non native ChaCha20 class. Id prefer to keep it such that the classes can be more easily diffed but made this change anyway. If its ok id like to change it back to match the non native class for maintenance reasons.

Reply:
That's a reasonable plan, but I see many other unnecessary differences between `NativeChaCha20Cipher` and `ChaCha20Cipher`. Please create an issue to eliminate as many of those differences as possible.

Reply:
I opened issue https://github.ibm.com/runtimes/jit-crypto/issues/362 to track these future efforts to sync up this code further.

Reply:
Ideally that issue would exist in a public repository, either this one or in eclipse-openj9/openj9.